eos/effects/industrialcoreeffect2.py

```diff
@@ -1,29 +1,100 @@
 # industrialCoreEffect2
 #
 # Used by:
-# Variations of module: Industrial Core I (2 of 2)
+# Module: Industrial Core I
 type = "active"
 runTime = "early"
 
 
-def handler(fit, module, context):
-    return
-
-    # @todo: finish this
+def handler(fit, src, context):
     fit.extraAttributes["siege"] = True
-    fit.ship.boostItemAttr("maxVelocity", module.getModifiedItemAttr("speedFactor"))
-    fit.ship.multiplyItemAttr("mass", module.getModifiedItemAttr("siegeMassMultiplier"))
-
-
-    fit.modules.filteredItemIncrease(lambda mod: mod.item.group.name in
-                                                 (
-                                                     "Command Burst",
-                                                 ) and
-                                                 mod.item.requiresSkill in
-                                                 (
-                                                     "Shield Command",
-                                                 ),
-                                     "warfareBuff4Multiplier",
-                                     module.getModifiedItemAttr("shipBonusORECapital3"),
-                                     skill="Capital Industrial Ships",
-                                     )
+    fit.ship.boostItemAttr("maxVelocity", src.getModifiedItemAttr("speedFactor"))
+    fit.ship.multiplyItemAttr("mass", src.getModifiedItemAttr("siegeMassMultiplier"))
+    fit.ship.multiplyItemAttr("scanResolution", src.getModifiedItemAttr("scanResolutionMultiplier"))
+
+    #  Remote Shield Repper Bonuses
+    fit.modules.filteredItemBoost(lambda mod: mod.item.requiresSkill("Capital Shield Emission Systems"),
+                                  "duration",
+                                  src.getModifiedItemAttr("industrialCoreRemoteLogisticsDurationBonus"),
+                                  )
+    fit.modules.filteredItemBoost(lambda mod: mod.item.requiresSkill("Capital Shield Emission Systems"),
+                                  "capacitorNeed",
+                                  src.getModifiedItemAttr("industrialCoreRemoteLogisticsRangeBonus"),
```

Comment:
>this should be industrialCoreRemoteLogisticsDurationBonus, not the range bonus

The `src.attribute` is correct.  The module attribute is not, copy/pasta error from local repper.

Fixed (soon to be submitted).

>You're also missing two effect related to this: maxRange and falloffEffectiveness, using the industrialCoreRemoteLogisticsRangeBonus attribute

How do you know that the range bonus effects `falloffEffectiveness`?